hirte.spec.in

```diff
@@ -103,6 +103,8 @@ Requires: selinux-policy >= %{_selinux_policy_version}
 
 Requires(post): policycoreutils
 
+%global selinuxtype	targeted
```

Comment:
Hmm, I somehow missed that :-( even though it's mentioned in selinux packaging guidelines:

https://fedoraproject.org/wiki/SELinux_Policy_Modules_Packaging_Draft#Building_the_Policy_Modules

So is `targeted` enough or do we need to add also `mls` to the list?

Reply:
> So is targeted enough or do we need to add also mls to the list?

That's why I wanted to get input from @rhatdan 

> Also as this is global option, could you please move it to the beginning of the spec file?

Don't you think it's more readable when it's defined next to the sub-package it belongs to?

Reply:
Let's leave it as targeted.  We don't plan on supporting any other form of SELinux policy especially not MLS.